src/hayhooks/server/app.py

```diff
@@ -35,9 +42,9 @@ def create_app() -> FastAPI:
 @app.get("/")
 async def root():
     return {
-        "swagger_docs" : "http://localhost:1416/docs",
-        "deploy_pipeline" : "http://localhost:1416/deploy",
-        "draw_pipeline" : "http://localhost:1416/draw/{pipeline_name}",
-        "server_status" : "http://localhost:1416/status",
-        "undeploy_pipeline" : "http://localhost:1416/undeploy/{pipeline_name}",
+        "swagger_docs": "http://localhost:1416/docs",
```

Comment:
Ok, so no router for docs? Comment maybe?

Reply:
I was thinking to remove that root route in another PR. There's no need of it actually - docs will be always available at:

- http://localhost:1416/docs (swagger)
- http://localhost:1416/redoc (redocly)

this route was probably added to avoid returning a 404 when hitting `/`, but there's no need to have it I guess.